newIDE/app/src/ExpressionAutocompletion/index.js

```diff
@@ -21,8 +21,11 @@ import { getVisibleParameterTypes } from '../EventsSheet/ParameterFields/Generic
 
 type BaseExpressionAutocompletion = {|
   completion: string,
+  replacementStartPosition: integer,
```

Comment:
In the IDE with Flow, we don't do the distinction between integer/float => `number`